pkg/dataloader/featuregateloader/featuregateloader.go

```diff
@@ -75,21 +66,16 @@ func (l *FeatureGateLoader) Errors() []error {
 	return l.errs
 }
 
-func (l *FeatureGateLoader) getTargetReleases() ([]string, error) {
+func (l *FeatureGateLoader) getTargetReleases() []string {
 	var targetReleases []string
-	releases, err := api.GetReleasesFromBigQuery(context.Background(), l.bqc)
-	if err != nil {
-		return nil, errors.Wrapf(err, "error querying releases from bq")
-	}
-
-	for _, release := range releases {
+	for _, release := range l.releaseConfigs {
 		if release.Capabilities[v1.FeatureGatesCap] {
```

Comment:
should this also filter out any with a future DevelStartDate? alternatively, we could just not add the `featureGates` capability until the branch occurs. either way we need a DB update